_benchmark/user-guide/understanding-workloads/anatomy-of-a-workload.md

```diff
@@ -0,0 +1,790 @@
+-  `document-count`: The number of documents in the `source-file`, which determines which client indexes correlate to which parts of the document corpus. Each N client receives an Nth of the document corpus. When using a source that contains a document with a parent-child relationship, specify the number of parent documents.
+- `uncompressed-bytes`: The size, in bytes, of the source file after decompression, indicating how much disk space the decompressed source file needs.
+- `compressed-bytes`: The size, in bytes, of the source file before decompression. This can help you assess the amount of time needed for the cluster to ingest documents.
+
+### Operations
+
+The `operations` element lists the OpenSearch API operations performed by the workload. For example, you can set an operation to `create-index`, an index in the test cluster to which OpenSearch Benchmark can write documents. Operations are usually listed inside of `schedule`.
+
+### Schedule
+
+The `schedule` element contains a list of actions and operations that are run by the workload. Operations run according to the order in which they appear in the `schedule`. The following example illustrates a `schedule` with multiple operations, each defined by its `operation-type`:
+
+```json
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": 5000
+      },
+      "warmup-time-period": 120,
+      "clients": 8
+    },
+    {
+      "operation": {
+        "name": "query-match-all",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+According to this schedule, the actions will run in the following order:
+
+1. The `create-index` operation creates an index. The index remains empty until the `bulk` operation adds documents with benchmarked data.
+2. The `cluster-health` operation assesses the health of the cluster before running the workload. In this example, the workload waits until the status of the cluster's health is `green`.
+   - The `bulk` operation runs the `bulk` API to index `5000` documents simultaneously.
+   - Before benchmarking, the workload waits until the specified `warmup-time-period` passes. In this example, the warmup period is `120` seconds.
+5. The `clients` field defines the number of clients that will run the remaining actions in the schedule concurrently.
```

Comment:
Could you update this line to be step 3 and the next line to be step 4?

Reply:
` will run the remaining actions in the schedule concurrently.` -> ` will run the bulk indexing operation concurrently.`

Reply:
@Naarcha-AWS To clarify point 3, could we rephrase ` will run the remaining actions in the schedule concurrently.` -> ` will run the bulk indexing operation concurrently.`